training/DeepSpeed-Domino/domino/training.py

```diff
@@ -109,6 +111,11 @@ def setup_model_and_optimizer(base_model,
     optimizer = get_megatron_optimizer(models, no_wd_decay_cond, scale_lr_cond)
     opt_param_scheduler = get_optimizer_param_scheduler(optimizer)
 
+    if args.load is not None:
+        args.iteration = load_checkpoint(model, optimizer, opt_param_scheduler)
+    else:
+        args.iteration = 0
+
     args.iteration = 0
```

Comment:
Delete this line. Otherwise, the `args.iteration` is wrong if the checkpoint is successfully loaded.

Reply:
Also, do we need to change `args.train_iters`?  `args.iteration` may be larger than `args.train_iters` if there is a checkpoint.